From 37694160594aeaea59b4a0e434344b1dd1962979 Mon Sep 17 00:00:00 2001
From: Yu Watanabe <watanabe.yu+github@gmail.com>
Date: Sun, 26 Mar 2023 09:44:48 +0900
Subject: [PATCH] hwdb: drop redundant entry

Follow-up for ff155c81620bc4e1ccce27d4d5194707048d8143.

The entry already matches with the generic rule defined at the beginning
of the hwdb file.

(cherry picked from commit 596ac132fb13943a291fb4150c234a1f856e7e8f)

Related: RHEL-5950
---
 hwdb.d/70-touchpad.hwdb | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/hwdb.d/70-touchpad.hwdb b/hwdb.d/70-touchpad.hwdb
index a87b91ffd2..262bca3a19 100644
--- a/hwdb.d/70-touchpad.hwdb
+++ b/hwdb.d/70-touchpad.hwdb
@@ -51,13 +51,6 @@ touchpad:usb:v05acp0265:*
 touchpad:usb:v044ep1221:*
  ID_INPUT_TOUCHPAD_INTEGRATION=external
 
-###########################################################
-# Lenovo
-###########################################################
-# Thinkpad L14 Gen1 (AMD)
-touchpad:usb:v06cbp0081:*
- ID_INPUT_TOUCHPAD_INTEGRATION=internal
-
 ###########################################################
 # Logitech
 ###########################################################
